docs/guides/developer/docs/api/base-api.md

```diff
@@ -6,14 +6,23 @@ In order to assess key characteristics of the API and to test general connectivi
 
 ### GET /api
 
-Returns key characteristics of the API such as the server name and the default version.
+Returns key characteristics of the API such as the Application API base URL and the default version.
 
 __Response__
 
-`200 (OK)`: The api information is returned.
+`200 (OK)`: The api information is returned as a JSON object containing the following fields:
+
+Field     | Type                       | Description
+:---------|:---------------------------|:-----------
+`version `| [`string`](types.md#basic) | Default version of the External API
+`url`     | [`string`](types.md#basic) | Base URL clients shall use to communicate with the External API
+
+
+__Example__
+
 ```
 {
-  "url": "https:\/\/api.opencast.org",
+  "url": "https:\/\/api.opencast.org\/api",
```

Comment:
Just double-checked this: The default configuration does indeed return this value.